src/s1a_portapotty.tw2

```diff
@@ -187,7 +187,7 @@ Steadying myself, I stood up, tested my footing, and glanced around. Out of the
 ::Outside - Note
 The note was signed, but I already knew who wrote it by the loopy lettering of the purple gel-pen on the paper.
 
-<div style="padding-left: 2em; font-style: italic;">Hey,
+[(live: 1s)[(if: time > 1s)[$dissolve[<div style="padding-left: 2em; font-style: italic;">Hey,
```

Comment:
This `live:` is largely meant to solve the issue of anything wrapped in a `<div>` displaying immediately, but I think the delay also helps pace out the reveal nicely.

Reply:
I like the delay here, but having the whole letter pop in still feels a little jarring. I'd love to see the letter itself broken up into smaller reveals with more live: macros. But, it's not essential for this PR.